src/pat/validation/validation.js

```diff
@@ -163,6 +166,18 @@ define([
                     constraint.numericality.onlyInteger = true;
                 }
             }
+
+            // Handle fields equality
+            if (opts.equality && input.value) {
```

Comment:
I think the ``&& input.value`` check should be removed. You probably still want to check for equality if no value was given. Maybe instead of the ``&& input.value`` we should enforce presence via the ``required`` attribute?

Reply:
I removed the check